ui/src/data_explorer/components/MeasurementList.js

```diff
@@ -111,15 +122,34 @@ const MeasurementList = React.createClass({
       <div className="query-builder--list">
         {measurements.map(measurement => {
           const isActive = measurement === this.props.query.measurement
+          const numTagsActive = Object.keys(this.props.query.tags).length
+
           return (
-            <div
-              className={classNames('query-builder--list-item', {
-                active: isActive,
-              })}
-              key={measurement}
-              onClick={_.wrap(measurement, this.props.onChooseMeasurement)}
-            >
-              {measurement}
+            <div key={measurement}>
+              <div className={classNames('query-builder--list-item', {active: isActive})}>
+                <div className="query-builder--caret icon caret-right"></div>
```

Comment:
i expect to be able to click the caret to expand as well, but this doesn't work-- only clicking the <span> below works.